File: 100-Days-Series/Day 70/ArrayTarget.java
public class ArrayTarget {
    // Funtion to find two numbers that add up to a specific target
    public static void getFind(int arr[],int target){
        for(int i=0;i<arr.length;i++){
            for(int j=i+1;j<arr.length;j++){
                if(arr[i]+arr[j]==target){
                    System.out.println("Indices : "+i+", "+j);
                    return;
                }
            }
        }
        System.out.println("No Solution Found");
    }
    // Main Method
    public static void main(String [] args){
        int []arr={2,5,6,8,10,23};
        int target=11;
        getFind(arr, target);
    }

}

File: 100-Days-Series/Day 70/ArrayTarget.cpp
#include<iostream>
using namespace std;
// Function to find two numbers that add up to a specific target
void getFind(int arr[],int arrsize,int target){
    for(int i=0;i<arrsize;i++){
        for(int j=i+1;j<arrsize;j++){
            if(arr[i]+arr[j]==target){
                cout<<"\nIndices : "<<i<<", "<<j;
                return;
            }
        }
    }
    cout<<"\nNo Solution Found!";
}
// Let's Start Here
int main(){
    int arr[]={2,3,10,12};
    int target=14;
    int size=sizeof(arr)/sizeof(arr[0]);
    getFind(arr,size,target);
}